src/components/item/form/EditModalWrapper.tsx

```diff
@@ -84,34 +81,28 @@ const EditModalWrapper = ({
         payload: { error: new Error(FAILURE_MESSAGES.UNEXPECTED_ERROR) },
       });
     } else {
-      editItem({ id: item.id, ...updatedProperties });
+      editItem({
+        id: updatedItem.id,
+        name: updatedItem.name,
+        description: updatedItem.description,
+        // todo: find a fix since the folder has an invalid extra
```

Comment:
Report in an issue?

Reply:
This comment was out of date, the fix is just below :)